src/flag_gems/runtime/backend/_cambricon/ops/div.py

```diff
@@ -38,6 +38,10 @@ def true_div_func_scalar_tensor(x, y):
 def true_divide(A, B):
     logger.debug("GEMS_CAMBRICON TRUE_DIVIDE")
     if isinstance(A, torch.Tensor) and isinstance(B, torch.Tensor):
+        if A.shape != B.shape:
```

Comment:
Why do we need this piece of code?
The `@pointwise_dynamic` decorator already includes the `broadcast` functionality, so it seems redundant here. Could you clarify the purpose?